docs/cep/source/stream-source.md

```diff
@@ -59,6 +63,10 @@ The following parameters are used to configure a stream definition.
 | num.listener.threads | The number of listener threads. | 1 | INT | Yes |
 | num.io.threads | The number of listener threads. | 1 | INT | Yes |
 | receiver.queue.size | The number of messages accumulated by a consumer before an application calls. | 1000 | INT | Yes |
+| stream.url | The URL of the Pulsar broker, e.g. pulsar_ssl://my-broker:6651. | NULL | STRING | Yes |
```

Comment:
The mention of Pulsar in this and the next line seem out of place. I have no idea how I would find any of the Pulsar things in our setup.

Reply:
@diana-macrometa - Here this is valid. We didn't have an ability to access external Pulsar cluster in our stream sources and sinks till now but Intel could be one such customer.